Makefile

```diff
@@ -7,21 +7,27 @@ help:
 	@echo "clean				Remove artifacts and standardize repo."
 
 # CORE ]------------------------------------------------------------------------
-test:
-	black --check .
+venv: venv/bin/activate
```

Comment:
Is there a reason all of this shouldn't just be in the `deps` rule?

Reply:
thanks for the review! yep, you're right, as deps literally does the exact same thing.

Reply:
i checked this one and it actually makes a difference. by calling `deps`, you tell make to do something that is not attached to any file (they called it a `phony`)

when having a rule to make an actual file, you will only executed `deps` (or `venv/bin/activate`) if this target is not yet present.

this has the neat side-effect that subsequent calls of everything that require the virtual environment are very fast.

Reply:
turns out that i got the chaining of commands wrong. fixed now, same functionality :)